sdk/storage/storage-queue/CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Release History
 
+## 12.0.2 (Unreleased)
+
+- Share a single httpClient among clients. [PR #6657](https://github.com/Azure/azure-sdk-for-js/pull/6657)
+  Currently clients share their parent client's pipeline. But when the pipeline doesn't have a httpClient specified, a `DefaultHttpClient` will be created for each of the clients. This change create a default http client instance to share among all clients if none is specified.
```

Comment:
This still doesnt clarify to the end user as to how sharing of clients helps them. You mentioned something about the keepAlive feature, can we use that here?

Reply:
Hope it is better now.